test/pylib/cpp/item.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+import os
+import string
+from pathlib import Path
+from typing import Sequence, Any, Iterator
+
+import pytest
+from _pytest._code.code import TerminalRepr, ReprFileLocation
+from _pytest._io import TerminalWriter
+
+from test.pylib.cpp.boost import BoostTestFacade, BoostTestFailure, BoostFailureError
+
+
+def get_code_context_around_line(filename: str, linenum: int) -> list[str]:
```

Comment:
this is a problem of putting 3rd party code into repo. now it is our code. so we need docstrings at least for everything(and it should be separate commit )